src/components/tabs/js/tabsController.js

```diff
@@ -395,7 +395,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
           isActive:     function () { return this.getIndex() === ctrl.selectedIndex; },
           isLeft:       function () { return this.getIndex() < ctrl.selectedIndex; },
           isRight:      function () { return this.getIndex() > ctrl.selectedIndex; },
-          shouldRender: function () { return !ctrl.noDisconnect || this.isActive(); },
+          shouldRender: function () { return ctrl.noDisconnect || this.isActive(); },
```

Comment:
I am not up to date on this component, but md-no -disconnect was used in the past when the scope was disconnected by default.

Sent from my iPhone

> On Mar 21, 2016, at 12:27 PM, Paul Gschwendtner <notifications@github.com> wrote:
> 
> In src/components/tabs/js/tabsController.js:
> 
> > @@ -395,7 +395,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
> >            isActive:     function () { return this.getIndex() === ctrl.selectedIndex; },
> >            isLeft:       function () { return this.getIndex() < ctrl.selectedIndex; },
> >            isRight:      function () { return this.getIndex() > ctrl.selectedIndex; },
> > -          shouldRender: function () { return !ctrl.noDisconnect || this.isActive(); },
> > +          shouldRender: function () { return ctrl.noDisconnect || this.isActive(); },
> I'm wondering, why there is another attribute md-no-disconnect, which disabled the disconnectfunctionality.
> 
> So that means, that we have two attributes (which are both reversed)
> 
> md-enable-disconnect (Documented)
> md-no-disconnect (Not documented)
> @robertmesserle Do you know anything about that?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>